addons/l10n_vn_edi_viettel/views/account_move_views.xml

```diff
@@ -18,6 +18,8 @@
                       invisible="move_type not in ['out_invoice', 'out_refund'] or country_code != 'VN'">
                     <group>
                         <field name="l10n_vn_edi_replacement_origin_id" invisible="1"/>
+                        <field name="l10n_vn_edi_reversed_entry_invoice_number" invisible="1"/>
+                        <field name="l10n_vn_edi_replacement_origin_invoice_number" invisible="1"/>
```

Comment:
```suggestion
                        <field name="l10n_vn_edi_reversed_entry_invoice_number" invisible="1"/>
```
Note that starting in I believe saas-17.3, adding invisible fields that are only used in invisible/required/... is no longer needed.
As there are others like that I'll take a look to clean them up later so no need for you to do anything on that side.

Reply:
Nice